helm/prometheus-rules/templates/alerting-rules/up.all.rules.yml

```diff
@@ -76,3 +71,24 @@ spec:
         severity: page
         team: atlas
         topic: observability
+
+    - alert: KubeStateMetricsNotRetrievingMetrics
```

Comment:
What's the difference with KubeSecretMetricMissing and KubeStateMetricsSlow?

Reply:
Also we have a KSMdown inhibition :)

Reply:
From my own understanding, `KubeStateMetricsSlow` is monitoring the response time from KSM to make sure it doesn't take too long to retrieve metrics while `KubeStateMetricsNotRetrievingMetrics` is making sure that there are actually metrics retrieved by KSM

But maybe we can consider that if KSM is taking too long to retrieve metrics then it means that it's unable to retrieve it and thus we can get rid of `KubeStateMetricsNotRetrievingMetrics` to only rely on `KubeStateMetricsSlow`

Reply:
Sure about the slow one but is this alert not the same as those https://github.com/giantswarm/prometheus-rules/blob/737a5e989ff13a86f868eb825ea0e7d71e8842fd/helm/prometheus-rules/templates/alerting-rules/kube-state-metrics.rules.yml#L31?

Also maybe we should regroup all KSM related alerts to this file

Reply:
Not sure, is `kube_configmap_created` a random metric normally retrieved by KSM that we check to make sure KSM is retrieving metrics in general ? 

In that case I'd prefer also keeping the new alert because its name is more straightforward about its usage and moreover it indicates that KSM as a whole is not able to retrieve metrics while the `KubeConfigMapCreatedMetricMissing` would indicate that KSM is not able to retrive metrics from a particular instance

Reply:
Well kube_configmap_created is the metric exposed by KSM when access the apiserver but we can have both if you prefer

Reply:
I guess it would make sense to have both yes